concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -1,27 +1,31 @@
 import mock
 
-from django.test    import override_settings
+from django.test import override_settings
 
 from conductor.models import BlenderSubtaskDefinition
 from core.tests.utils import ConcentIntegrationTestCase
 from utils.constants import ErrorCode
 from utils.helpers import get_storage_result_file_path
 from utils.helpers import get_storage_source_file_path
-from utils.testing_helpers  import generate_ecc_key_pair
+from utils.testing_helpers import generate_ecc_key_pair
 from ..constants import VerificationResult
-from ..tasks        import blender_verification_order
+from ..tasks import blender_verification_order
 
 
 def mock_store_file_from_response_in_chunks_raise_exception(_response, _file_path):
     raise OSError
 
 
-(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
+def mock_unpack_archive_raise_exception(_file_path):
```

Comment:
it is used only once and might be inlined (just add `side_effect=OSError` in patch),
BTW: In the new patches it would be good to add `autospec=True` argument ;) 
https://github.com/Code-Poets/coding-style/wiki/TECH-TALK-3---MOCKOWANIE#autospec---czego-chcie%C4%87-wi%C4%99cej

Reply:
:+1:

Reply:
Please add `autospec=True` argument to `patch(...)` calls

Reply:
OK, it seems that there is an old BUG in python (https://bugs.python.org/issue28380) which was closed without merge. I opend new one (https://bugs.python.org/issue33643).
As for now, the workoround is to use `assert_called_once_with` (which __is__ supported) on mocks that were autospecced